### PR TITLE
Add new 3.12 uniques

### PR DESCRIPTION
**Pull request by ppoelzl**

### CODE REVIEW

File: Data/New.lua [hunk 1]
```diff
@@ -3,8 +3,19 @@
...
+Corpsewalker
+Carnal Boots
+League: Heist
+Requires Level 55, 52 Dex, 52 Int
+Implicits: 0
+Triggers Level 20 Corpse Walk when Equipped
+94% increased Evasion and Energy Shield
+25% increased Movement Speed
+40% increased Damage if you have Consumed a corpse Recently
+For each nearby corpse, Regenerate 0.25% Life per second, up tp 3%
```
Comment (Nightblade) [inline Data/New.lua:17]: "tp" should be "to"

**Comment by Nightblade** [comment]
How do I add more items to this PR?  Or do I just make a new PR?

File: Data/New.lua (hunk 1 above)
Comment (ppoelzl) [reply Data/New.lua:17]: Thanks!

**Comment by ppoelzl** [comment]
Which ones do you want to add? If so, just make a new PR for them. I would be glad to have some assistance with this.

**Comment by Nightblade** [comment]
The ones that have been "spoilt" so far.  Just looking to fill a few minutes here and there :-)

**Comment by Nightblade** [comment]
I've included the corrected Corpsewalker in PR  #1391, so you can probably close this one.

PS - Sorry for the mess, I'm a total Github noob!  :-)

**Comment by Nightblade** [comment]
Did I make a mistake in my PR?

**Review by LocalIdentity — APPROVED** [review]

**Pull request merged**